jekyll/_cci2/faq.md

```diff
@@ -299,8 +299,9 @@ non-OSS projects. This includes activities such as:
 
 - Commits from users that trigger builds, including PR Merge commits.
 - Re-running jobs in the CircleCI web application, including [SSH debug]({{ site.baseurl }}/2.0/ssh-access-jobs).
-- Approving [manual jobs]({{ site.baseurl }}/2.0/workflows/#holding-a-workflow-for-a-manual-approval) (approver will be considered the actor of all downstream
-  jobs).
+  - Approving [manual jobs]({{ site.baseurl }}/2.0/workflows/#holding-a-workflow-for-a-manual-approval) (approver will be considered the actor of all downstream jobs).
+  - Using scheduled workflows
+  - Using the machine executor
```

Comment:
Why do we call out `machine executor` here? Also, we'll probably want to include `Guest user`s, i.e. a user who reruns a build but doesn't have an equivalent Github user.

Reply:
I think I misunderstood the request, going to DM and clear up.

Reply:
@emmabukacek -- For `guest user`, could we be clearer with this language? Customers see multiple "guest users" on the account but do not understand what this is connected to. Is it API calls, scheduled workflows, users from open source projects?